Tests/UtilityTests/ArgumentParserTests.swift

```diff
@@ -367,6 +368,21 @@ class ArgumentParserTests: XCTestCase {
         }
     }
 
+    func testPathArgument() {
+        // relative path
```

Comment:
better comments appreciated! foe e.g.: `// Test that relative path is resolved.`